GOVERNANCE.md

```diff
@@ -1,25 +1,100 @@
-| Needs Review | Issues that are ready for review. Code complete, pending feedback. | | After an issue is reviewed, the issue will either be approved and merged or it will require further changes. If the issue is approved and merged, the issue should be moved to **Done**. If the issue requires further changes, it can remain in **Needs Review** until approved. |
-| Done | Issues that have been approved and are waiting to be released. | | After this work is included in a published release, the issue can be moved to **Closed**.
-| Closed | Issues that have been released, resolved or deemed not product viable. | * `in-progress` (remove) | |
+# Carvel Governance
+This document defines the project governance for Carvel.
 
+---
+# Overview
+Carvel, an open source project, is committed to building an open, inclusive, productive and self-governing open source community focused on building high quality, reliable, single-purpose, composable tools that aid in your application building, configuration, and deployment to Kubernetes. The community is governed by this document with the goal of defining how the community should work together to achieve this goal.
+
+---
+# Code Repositories
+The following code repositories are governed by the Carvel community and maintained under the `vmware-tanzu\carvel` organization.
+
+* [Carvel](https://github.com/vmware-tanzu/carvel): Main Carvel Repo
+* [Carvel-ytt](https://github.com/vmware-tanzu/carvel-ytt): Template and overlay Kubernetes configuration via YAML structures, not text documents
+* [Carvel-kapp](https://github.com/vmware-tanzu/carvel-kapp): Install, upgrade, and delete multiple Kubernetes resources as one "application"
+* [Carvel-kbld](https://github.com/vmware-tanzu/carvel-kbld): Build or reference container images in Kubernetes configuration in an immutable way
+* [Carvel-imgpkg](https://github.com/vmware-tanzu/carvel-imgpkg): Bundle and relocate application configuration (with images) via Docker registries
+* [Carvel-kapp-controller](https://github.com/vmware-tanzu/carvel-kapp-controller): Capture application deployment workflow in App CRD. Reliable GitOps experience powered by kapp
+* [Carvel-vendir](https://github.com/vmware-tanzu/carvel-vendir): Declaratively state what files should be in a directory
+
+**Experimental:**
+* [kwt](https://github.com/vmware-tanzu/carvel-kwt)
+* [terraform-provider-carvel](https://github.com/vmware-tanzu/terraform-provider-carvel)
+* [carvel-secretgen-controller](https://github.com/vmware-tanzu/carvel-secretgen-controller)
+
+**Installation:**
+* [homebrew-carvel](https://github.com/vmware-tanzu/homebrew-carvel)
+* [carvel-docker-image](https://github.com/vmware-tanzu/carvel-docker-image)
+* [asdf-carvel](https://github.com/vmware-tanzu/asdf-carvel)
+* [carvel-setup-action](https://github.com/vmware-tanzu/carvel-setup-action)
+
+**Plugins:**
+* [ytt.vim](https://github.com/vmware-tanzu/ytt.vim)
+
```

Comment:
looks like we're missing one:
`[vscode-ytt](https://github.com/vmware-tanzu/vscode-ytt)`

Reply:
This was missing from the README too, so I just added it there.